dns/query.py

```diff
@@ -1089,11 +1098,20 @@ def tls(
         where, port, source, source_port
     )
     if ssl_context is None and not sock:
-        ssl_context = ssl.create_default_context()
+        cafile: Optional[str] = None
+        capath: Optional[str] = None
+        if isinstance(verify, str):
+            if os.path.isfile(verify):
+                cafile = verify
+            elif os.path.isdir(verify):
+                capath = verify
```

Comment:
Should this raise an exception if it's neither a file or directory?  Not sure if ignoring is the best option.

Reply:
good idea, see updated patch